### PR TITLE
{compiler} [dummy] GCC/6.2.0 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
Currently, with isl disabled.

Don't merge until isl works too.

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 7 out of 7 (8 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/09c61c063a04223ba8807c24faceb77a for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2491.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/0746aad7211fe94e39198ff43ed3628c for a full test report.

File: easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    '4c175f86e11eb32d8bf9872ca3a8e11d',     # gmp-6.1.1.tar.bz2
+    'b8a2f6b0e68bef46e53da2ac439e1cf4',     # mpfr-3.1.4.tar.gz
+    'd6a1d5f8ddea3abd2cc3e98f58352d26',     # mpc-1.0.3.tar.gz
+    'ac1f25a0677912952718a51f5bc20f32',     # isl-0.16.1.tar.bz2
+    '13a670dca515e905856f95d13697fcdd',     # mpfr-3.1.2-allpatches-20160522.patch
+]
+
+languages = ['c', 'c++', 'fortran']
+
+withisl = False
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb]: @wpoely86 please make this WIP until this is fixed?

Also, what's the problem here exactly?

Comment (wpoely86) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb]: fails to find the freshly build libraries going from stage 2 to 3.

Comment (fgeorgatos) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb]: I believe to have seen the same issue, you might as well not be alone in this

Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb]: No trace of similar issues in the GCC mailing list or something?

Comment (AFCJamie) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.1.0.eb]: We have experienced similar issues with 'Could not find a usable isl' when building GCC 6.1.0 with the field withisl set to True. A hand build of GCC 6.1.0 with ISL support seems okay though.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 7 out of 8 (8 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f3767d08791c6ec0097e5f3eff304941 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 7 out of 8 (8 easyconfigs in this PR)
node2609.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b13182fb33da9cddbd21ed632d4b6c9b for a full test report.

**Comment by wpoely86** [comment]
@boegel any reason why we don't check the checksum of patches in the tests?

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2609.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/33819d8825c3d33815643a6d2bed2e8d for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node801.raichu.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7cc45955f08313b5977ff14f8c2eb7b5 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2406.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/65f06938a42a5907232eec9795647dd4 for a full test report.

**Comment by wpoely86** [comment]
@boegel fixed and good to go.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2008.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5639220a03dd720d76ea5610a1bdf8b1 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 8 (8 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/366325f3f5092f6e4fc2408c88545a93 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 8 (8 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3bf258dca873b541847187363e748880 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 8 (8 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e89e9823e66f52b50315591e22f1bbe4 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 1 out of 8 (8 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/48cc766865ec8191291ee3362833fad2 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/9d9ed50ddf3eac3f922732db2b4867f5 for a full test report.

File: easybuild/easyconfigs/b/binutils/binutils-2.27-GCCcore-6.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '6.2.0'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [
+    ('flex', '2.6.0'),
+    ('Bison', '3.0.4'),
+    # zlib required, but being linked instatically, so not a runtime dep
```
Comment (boegel) [inline easybuild/easyconfigs/b/binutils/binutils-2.27-GCCcore-6.2.0.eb]: @wpoely86 you managed to drop a space here: `instatically` => `in statically`

File: easybuild/easyconfigs/b/binutils/binutils-2.27.eb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SOURCE]
+
+builddependencies = [
+    ('flex', '2.6.0'),
+    ('Bison', '3.0.4'),
+    # zlib required, but being linked instatically, so not a runtime dep
```
Comment (boegel) [inline easybuild/easyconfigs/b/binutils/binutils-2.27.eb]: @wpoely86 same here: `in statically`

File: easybuild/easyconfigs/g/GCCcore/mpfr-3.1.4-allpatches-20160804.patch [hunk 4]
```diff
@@ -0,0 +1,453 @@
+diff -Naurd mpfr-3.1.4-a/PATCHES mpfr-3.1.4-b/PATCHES
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/mpfr-3.1.4-allpatches-20160804.patch]: @wpoely86 include ref to source of this patch?

File: easybuild/easyconfigs/g/GCCcore/GCCcore-6.2.0-fix-find-isl.patch [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Don't use libmpc and libmpfr to find libisl
```
Comment (boegel) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.2.0-fix-find-isl.patch]: @wpoely86 please add yourself as an author

does it make sense to contribute this upstream?

Comment (wpoely86) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.2.0-fix-find-isl.patch]: yes, already looking into it.

Comment (wpoely86) [inline easybuild/easyconfigs/g/GCCcore/GCCcore-6.2.0-fix-find-isl.patch]: And it's already cryptographically signed with my key. Even better and just mention the author.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node864.raichu.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/be34406ccc30e97cf5d503915906d137 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2427.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e4c26adae9db54bbe563794d2e11c284 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2062.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d0be33eba8b91fb62eada74606eaffb8 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**